unix/content/unix.tex

```diff
@@ -50,6 +31,40 @@
   \end{itemize}
 \end{frame}
 
+\begin{frame}{Terminal-Emulatoren}
```

Comment:
Wirkt sehr nach WoT, vielleicht ein paar Screenshots von den verschiedenen optionen?

Inhaltlich aber ok.

Reply:
WoT?

Reply:
Wall of Text :)

Reply:
okay. vielleicht krieg ich es noch etwas schöner hin. werde minimum screenshots hinzufügen.

Reply:
Zieht sich im Moment durch alle unsere Folien durch, das ist der Nachteil wenn man Folien auch als Referenzen nutzt. 

Aber ein paar Bilder hier und da machen vieles Besser und auch die Folien unterscheidbarer

Reply:
ich kann ja dann auch verschiedene Beispiele in der Live-Demo kurz zeigen.
Gerade bei dem Unix Vortrag sind die Slides ja eher sekundär.

Reply:
Kannst ja auch den Pipboy aus der Fallout Reihe zeigen, als populäres Beispiel

Reply:
Ich war ja schon bei WoT raus, aber bei Gaming Insidern bin ich dann völlig am Ende haha. Ich werd mal recherchieren.